docs/user-guide/cli-configure-scs-on-headless-linux-os.md

```diff
@@ -58,42 +72,85 @@ When you are using SSH or TTY to log in to Linux, you can configure the Gnome ke
     fi
     ```
 
+    The first command launches DBus, which the Gnome keyring requires. The second command starts the keyring daemon so that it is ready to be used by Zowe CLI commands.
+
 4. Restart your computer.
 
-    Issue a Zowe CLI command that uses secure credentials to test that automatic unlock of the keyring works.
+    You have successfully completed the configuration to unlock the Gnome keyring automatically.
+
+5. Issue a Zowe CLI command that uses secure credentials to validate the automatic keyring unlock.
 
-## Configuring z/Linux
+## z/Linux operating systems
 
-The Secure Credential Store (SCS) does not contain the  native, pre-built binaries that are required to access the credential vault on z/Linux operating systems.
+### Configuring z/Linux
 
-Because the credential manager is now a built-in function of Zowe CLI, developers must build the credential mananger binaries on z/Linux systems during the Zowe CLI installation process.
+Zowe CLI does not contain the  native, pre-built binaries that are required to access the credential vault on z/Linux operating systems. Developers must build the credential manager binaries on z/Linux systems during the Zowe CLI installation process.
 
-The following steps describe how to install and build the credential store binaries on z/Linux (Red Hat Enterprise Linux (RHEL) and Ubuntu) systems.
+For instructions to set up the credential manager binaries for Red Hat Enterprise Linux (RHEL) V8.X and Ubuntu z/Linux systems, refer to this section. See [Configuring RHEL V7.X](#configuring-rhel-v7x) for instructions specific to that distribution.
```

Comment:
For consistency, how about changing second part to "For instructions specific to Red Hat Enterprise Linux (RHEL) V7.X, see ..."